### PR TITLE
select portis network before connect

### PR DESCRIPTION
**Pull request by chescalante**
https://user-images.githubusercontent.com/8353287/145475622-19971e71-6e1c-42dd-b679-c3562cc10c35.mov

### CODE REVIEW

File: src/lib/hardware-wallets.ts [hunk 1]
```diff
@@ -60,6 +65,18 @@ const TORUS_NETWORK_PARAMS: NetworkParamsOptions = {
   }
 }
 
+const PORTIS_NETWORK_PARAMS: NetworkParamsOptions = {
```
Comment (chescalante) [inline src/lib/hardware-wallets.ts:68]: should we rename this file or move portis config into another?

File: src/lib/networkOptionsTypes.ts [hunk 2]
```diff
@@ -1,11 +1,12 @@
 export type NetworkParams = {
```
Comment (ilanolkies) [inline src/lib/networkOptionsTypes.ts:1]:
```suggestion
export type NetworkParams = TorusNetworkParams | PortisNetworkParams
```

**Review by ilanolkies — CHANGES_REQUESTED** [review]
We need to update `sample/index.html` to explain how to use it. See https://github.com/rsksmart/rLogin/blob/develop/sample/front/index.html#L114

**Review by jessgusclark — APPROVED** [review]
Works good for me. I did submit [an issue with the disabled button](https://github.com/rsksmart/rLogin/issues/205) but that can be done in a different PR.

File: src/lib/networkOptionsTypes.ts (hunk 2 above)
Comment (chescalante) [reply src/lib/networkOptionsTypes.ts]: done!

**Review by ilanolkies — APPROVED** [review]

**Pull request merged**